packages/ap-frontend/docs/templatetags.rst

```diff
@@ -671,3 +671,28 @@ Usage:
 .. code-block:: html+django
 
     {% component "@alliancesoftware/ui" "NumberInput" default_value=widget.value|none_as_nan %}{% endcomponent %}
+
+
+``camelize``
+--------------------
+
+Recursively converts dictionary keys from ``snake_case`` to ``camelCase`` for passing props to the frontend
+
+Calls :meth:`~allianceutils.util.camelize <https://github.com/AllianceSoftware/django-allianceutils?tab=readme-ov-file#camelize>`_
```

Comment:
this is wrong format (see [output here](https://alliance-platform--52.org.readthedocs.build/projects/frontend/52/templatetags.html#camelize))

Should just be 

```
Calls `camelize <https://github.com/AllianceSoftware/django-allianceutils?tab=readme-ov-file#camelize>`_
```

Reply:
Fixed, thanks.